test/WebJobs.Script.Tests/Workers/Rpc/WebHostRpcWorkerChannelManagerTests.cs

```diff
@@ -86,6 +86,9 @@ public async Task CreateChannels_Succeeds()
             Assert.NotNull(initializedChannel);
             Assert.Equal(javaWorkerChannel.Id, initializedChannel.Id);
             Assert.Equal(_rpcWorkerChannelManager.GetChannels(RpcWorkerConstants.JavaLanguageWorkerName).Count(), 2);
+
+            // test case insensitivity
+            Assert.Equal(_rpcWorkerChannelManager.GetChannels("Java").Count(), 2);
```

Comment:
The WebHost manager was already case-insensitive, but added an explicit test just to prevent changes in the future.

https://github.com/Azure/azure-functions-host/blob/c52bb6f0f2a8eb31eebe0367fca3d38873ac10f9/src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs#L35